helm-charts/images/hub/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN pip install --no-cache git+https://github.com/yuvipanda/jupyterhub-configura
 RUN pip install --no-cache --upgrade jupyterhub-kubespawner==1.1.2
 
 # Brings in https://github.com/jupyterhub/oauthenticator/pull/498
-RUN pip install --no-cache --upgrade git+https://github.com/yuvipanda/oauthenticator@7f3fdc0a14d06f1a081c23b1ceb7060a940d11f8
+RUN pip install --no-cache --upgrade git+https://github.com/jupyterhub/oauthenticator@d189037fe6b57453a3ae733a60740c50f241e713
```

Comment:
@GeorgianaElena, can we create a follow-up issue to use a released version when that one is released? I have been bitten before when I used unreleased versions...